### DISCUSSION TITLE
Integrate Monailabel on OHIF viewer on google cloud platform

### DISCUSSION THREAD

**Discussion by risheetreddy** [Project-MONAI/MONAILabel#1306] [Q&A]
There is a requirement to integrate Ohif Dicom viewer with Monai-label in google cloud platform for automated image annotations

We have already deployed the OHIF Dicom viewer on GCP Cloud run and now we want to integrate Monai-label on that OHIF viewer for automated image annotations on GCP, is there a way to integrate on cloud?

**Comment by SachidanandAlle** [comment]
Which Ohif version?  If it's 2x.. then you can simply include monailabel plugin directory from plugin/Ohif GitHub repo into your Ohif build..

**Reply by risheetreddy** [reply]
Yes its 2x, but the issue is we need to access the data from the dicom store which is present in GCP not from the the local/remote dicom-web storage.

If we include that monailabel plugin on ohif build it is accessing from the local/remote dicom-web storage

**Comment by SachidanandAlle** [comment]
We already have someone from Google integrated this... for example this was added by one of them to support auth for GCP
https://github.com/Project-MONAI/MONAILabel/blob/main/monailabel/interfaces/app.py#L158-L161
